packages/desktop-client/src/components/accounts/MobileAccounts.js

```diff
@@ -50,8 +54,8 @@ function AccountCard({ account, updated, getBalanceQuery, onSelect }) {
       style={{
         flex: '1 0 auto',
         flexDirection: 'row',
-        backgroundColor: 'white',
-        boxShadow: `0 1px 1px ${colors.n7}`,
+        backgroundColor: theme.tableBackground,
+        boxShadow: `0 1px 1px ${theme.cardShadow}`,
```

Comment:
The new shadow is much stronger than before (on mobile - accounts list). Can we bump it back down for lightmode?

Reply:
The text color is also pretty different on mobile account view now.